Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.8
+
+ENV FLASK_APP=app.py
+
+ENV FLASK_RUN_HOST=0.0.0.0
+
+WORKDIR /app
+
+COPY . .
+
+# Install packages
+RUN pip3 install -r requirements.txt
+
+COPY .env.example .env
+
+RUN rm site.db
```

Comment:
hey @benji011  since we are already using env_file in the docker compose I dont think we need to copy the .env.example here , .env.example also does not have any relevant values

Reply:
Also could you send the PR to the development branch from where I will merge to master as I am following that flow .

Reply:
Rest other than that I see no issue.

Reply:
Thanks! I removed in https://github.com/DiptoChakrabarty/flask-online-store/pull/10/commits/e960eba72e788cf2f6948032ad1e12fa17610943